src/parser/view_transform.c

```diff
@@ -1557,6 +1561,32 @@ mq_remove_select_list_for_inline_view (PARSER_CONTEXT * parser, PT_NODE * statem
       /* just add one of integer value */
       new_select_list = pt_make_integer_value (parser, 1);
     }
+  else if (PT_SPEC_IS_DERIVED (spec) && spec->info.spec.derived_table_type == PT_IS_SUBQUERY)
+    {
+      spec->info.spec.referenced_attrs = new_select_list;
+
+      for (col = new_select_list; col; col = col->next)
+	{
+	  attr_order = pt_find_node_order (parser, spec->info.spec.as_attr_list, col);
+	  if (attr_order != -1)
+	    {
+	      nth_node = pt_get_node_from_list (subquery->info.query.q.select.list, attr_order - 1);
+	      assert (nth_node != NULL);
+
+	      if (PT_IS_ANALYTIC_NODE (nth_node))
+		{
+		  temp = nth_node->next;
+		  nth_node->next = NULL;
+
+		  parser_walk_tree (parser, nth_node, mq_update_node_order, spec, pt_continue_walk, NULL);
+		  nth_node->next = temp;
```

Comment:
```suggestion
		  /*
		   * e.g. drop table if exists ta, tb;
		   *      create table ta (ca int, cb int, cc int);
		   *      create table tb (ca int, cb int, cc int);
		   *
		   *      select b_cb, b_row_number
		   *        from ( select a.cb as a_cb,
		   *                      b.cb as b_cb,
		   *                      row_number () over (partition by b.cb order by b.cc) as b_row_number
		   *                 from ta a, tb b
		   *                where a.ca = b.ca
		   *                limit 1 ) t;
		   * 
		   *      new_select_list :
		   *                         t.b_cb, t.b_row_number
		   *
		   *      spec->info.spec.as_attr_list :
		   *                   a_cb,   b_cb,   b_row_number
		   *
		   *      Since b_ca does not exist in new_select_list, the indexes after b_ca are decremented.
		   *
		   *      spec->info.spec.derived_table->info.query.q.select.list :
		   *        - before : a.cb,   b.cb,   row_number() over (partition by 2 order by b.cc)
		   *        -  after : a.cb,   b.cb,   row_number() over (partition by 1 order by b.cc)
		   */
		  parser_walk_tree (parser, nth_node, mq_update_node_order, spec, pt_continue_walk, NULL);

		  nth_node->next = temp;
```